tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -33,6 +34,23 @@ const snackbar = () => $('#snackbar.active .snackbar-message');
 const snackbarMessage = async () => (await $('#snackbar.active .snackbar-message')).getText();
 const snackbarAction = () => $('#snackbar.active .snackbar-action');
 
+//Hamburguer menu
+//User settings
+const USER_SETTINGS = '=User settings';
+const UPDATE_PASSWORD = '=Update password';
+const EDIT_PROFILE = '=Edit user profile';
+// Feedback or Report bug
+const FEEDBACK_MENU = 'i.fa-bug';
```

Comment:
I think more specific the better
```suggestion
const FEEDBACK_MENU = '#header-dropdown i.fa-bug';
```